metagpt/tools/openai_text_to_embedding.py

```diff
@@ -43,12 +42,12 @@ class Config:
 
 
 class OpenAIText2Embedding:
```

Comment:
Is it possible to use OpenAI's aclient directly?

Reply:
The official website did not provide information, so I didn't dare to use it.